### ISSUE TITLE
Create the signup/login page

### ISSUE THREAD

**Issue by Aryan-Kumar-Thakur** [gdsc-jssstu/Programming_Community#5]
The page contain login and signup options for both the normal users and the core members. The required sections of the form is given in the UI/UX design.

Again you use demo url and images whenever needed.

Use the UI/UX Design for reference.

Make changes in signup_login.html , signup_login.css , signup_login.js for this page.

Request to assign the issue to avoid any conflict.

**Comment by Anshikaashuklaa** [comment]
Hi @Aryan-Kumar-Thakur Can you please assign this to me?

**Comment by Aryan-Kumar-Thakur** [comment]
> Hi @Aryan-Kumar-Thakur Can you please assign this to me?

Yeah sure, I will assign this to you. Fell free to generate a pull request once you done with your work. All the best 👍.

**Comment by Anshikaashuklaa** [comment]
Thankyou :)

**Comment by M27afk** [comment]
Any updates @Anshikaashuklaa ?
